admin/config.py

```diff
@@ -18,6 +18,7 @@ class Config:
     SQLALCHEMY_DATABASE_URI = get_connection_string()
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SQLALCHEMY_KEY = os.environ.get("SQLALCHEMY_KEY")
+    DEFAULT_ADMIN_USER = os.environ.get("DEFAULT_ADMIN_USER")
```

Comment:
Can DEFAULT_ADMIN_USER be removed from `grc/config.py`? grc app doesn't use it at all